opentelemetry-example/src/main/scala/zio/telemetry/opentelemetry/example/http/Client.scala

```diff
@@ -1,32 +1,29 @@
 package zio.telemetry.opentelemetry.example.http
 
-import sttp.client3._
-import sttp.client3.ziojson._
-import sttp.model.Uri
-import zio.telemetry.opentelemetry.example.config.AppConfig
 import zio._
+import zio.http.{ Header, Headers, Request, URL }
+import zio.json._
 import zio.telemetry.opentelemetry.example.Backend
+import zio.telemetry.opentelemetry.example.config.AppConfig
 
 case class Client(backend: Backend, config: AppConfig) {
 
   private val backendUrl =
-    Uri
-      .safeApply(config.backend.host, config.backend.port)
-      .map(_.withPath("status"))
+    URL
+      .decode(s"http://${config.backend.host}/${config.backend.port}")
```

Comment:
```
.decode(s"http://${config.backend.host}/${config.backend.port}")
```
Should be changed to (semicolon between host and port instead of a slash):
```
.decode(s"http://${config.backend.host}:${config.backend.port}")
```

Reply:
fixed